src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-redis.mdx

```diff
@@ -0,0 +1,174 @@
+
+<img
+  title="Redis OpenTelemetry quickstart dashboard"
+  alt="Image of the dashboard available through the Redis OpenTelemetry quickstart"
+  src={infrastructureRedisOTELDashboard}
+/>
+
+
+<Callout variant="tip">
+  If you're looking for help with other collector use cases, see the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository.
+</Callout>
+
+Complete the steps below to collect metrics from Redis:
+
+## Step 1: Sign up for your free account if you haven't already [#signup]
+
+<InlineSignup />
+
+## Step 2: Prerequisites [#prerequisites]
+
+The collector will use the [redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver) component (minimum version recommended v0.83.0), and it requires access to Redis through an endpoint.
+
+## Step 3: Install OpenTelemetry collector [#install-opentelemetry-collector]
+
+Download and install the OpenTelemetry collector following [OpenTelemetry docs](https://opentelemetry.io/docs/collector/getting-started/).
+
+You will need to install [OpenTelemetry Collector Contrib Distro](https://github.com/open-telemetry/opentelemetry-collector-releases/tree/main/distributions/otelcol-contrib) or other distribution including, at least, the following components:
+
+* [Redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/redisreceiver)
+* [OTLP exporter](https://pkg.go.dev/go.opentelemetry.io/collector/exporter/otlpexporter)
+* [Attribute processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/attributesprocessor/README.md)
+* [Resource processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourceprocessor/README.md)
+* [Resource detection processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/processor/resourcedetectionprocessor/README.md)
+
+## Step 4: Configure OpenTelemetry collector [#configure-opentelemetry-collector]
+
+Update or create a new file called `config.yaml` from the example below.
+
+Notice that, since no identificator is attached by the OTel receiver to redis metrics, an attribute processor is needed to decorate metrics with `server.address` and `server.port` metadata. 
+The attributes `server.address` and `server.port` are needed to be able to query and facet data differentiating between Redis servers and to create NewRelic entities. 
+
+Moreover, separate pipelines with different instances of `resource/redis_metrics` are needed in case multiple Redis servers are monitored from a single otel collector to decorate each instance with the corresponding metadata.
+
+
+Replace the `NEW_RELIC_LICENSE_KEY` in the file with your own values. For more info, see the [New Relic License key](/docs/apis/intro-apis/new-relic-api-keys/#license-key).
+
+```yaml
+receivers:
+  redis:
+    endpoint: "{redis_hostname}:{redis_port}"
```

Comment:
For the tech writers:

the customer needs to replace these with the corresponding values for redis_hostname and redis_port. What is the best way to make it clear in the YAML? 

In the other part of the docs I think we capitalize it like REDIS_HOSTNAME. Should we follow the same approach?

Reply:
We're starting to put customer-specific names for values inside `<BRACKETS>`. I'll do a sweep to add this to my batch of edits before merging. :)